index.html

```diff
@@ -218,9 +218,150 @@ <h2 class="section-title">About me</h2>
           <button class="button">Get my resume</button>
         </div>
       </section>
+      <form
+        id="fs-frm"
+        name="simple-contact-form"
+        accept-charset="utf-8"
+        action=""
+        method="post"
+      >
+        <fieldset id="fs-frm-inputs">
+          <input
+            type="text"
+            name="name"
+            class="data-input"
+            placeholder="First and Last"
+            required=""
+            maxlength="30"
+            minlength="5"
+          />
+          <input
+            type="email"
+            name="email"
+            class="data-input"
+            placeholder="email@domain.tld"
+            required=""
+            maxlength="30"
+            minlength="5"
+          />
+          <textarea
+            class="text-area"
+            rows="5"
+            name="message"
+            id="message"
+            placeholder="Hey I'd like to connect to..."
+            required=""
+            maxlength="500"
+            minlength="5"
+          ></textarea>
+          
+        
+          
+          
+          <input
+            type="hidden"
+            name="_subject"
+            id="email-subject"
+            value="Contact Form Submission"
+          />
+        </fieldset>
+        
+        <input type="submit" value="Get in touch" class="button" />
+      </form>
     </main>
     <footer>
-      <div></div>
+      <section class="container">
```

Comment:
- [ ] As part of the requirement for this project is to replicate exactly the Figma design, I think you need to improve on your button a little bit to match the Figma design. :wave: 
- Your button size is a bit smaller than the given design and also has a black border around it which is not in the given design. I suggest you look into this :+1:
-  Your form section is missing the text before the form as I circled in the screenshot below. Please go through the Figma design one more time and make try to make sure your project looks like the given design. :smile: 
- Your project is also missing a background image, I circled it in the screenshot below., I'll suggest you implement this too.
- [OPTIONAL] I noticed your project is missing the technologies list in the about section which was a requirement for the previous milestone. I'll suggest you go through your code history and check if you removed this section by mistake so that you can reimplement it. :wave: :+1: 

- - YOUR DESIGN:
![Selection_030](https://user-images.githubusercontent.com/107632780/235796765-3c77b973-14f1-43af-b648-7313bc2f7732.png)

- - FIGMA DESIGN:
![Selection_029](https://user-images.githubusercontent.com/107632780/235796822-fddd765b-7e28-4ec9-8cc9-8573e300b2a1.png)